dags/openshift_nightlies/config/install/aws-arm/ovn-cp.json

```diff
@@ -7,26 +7,26 @@
     "openshift_worker_count": 3,
     "openshift_host_prefix": "22",
     "openshift_network_type": "OVNKubernetes",
-    "openshift_master_instance_type": "m6g.4xlarge",
+    "openshift_master_instance_type": "r6g.4xlarge",
```

Comment:
I had lot of problems creating clusters with r6g that went away with m6g.

Reply:
Ok, rolling it back to m6g